### PR TITLE
added Carousel

### PR DESCRIPTION
**Pull request by lmeeta**
- Created basic carousel with dummy images.
- Added react router.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,44 +1,19 @@
...
-              );
-            })}
-          </tbody>
-        </table>
-      ) : (
-        "Enter a table number"
-      )}
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/table" element={<TableUsingMap />} />
```
Comment (lade6501) [inline src/App.jsx]:
```suggestion
        <Route path="/tableusingmap" element={<TableUsingMap />} />
```
`table` is html tag

File: src/Daily Practice/Home/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import "./Home.css";
+import { Link } from "react-router-dom";
+
+const Home = () => {
+  return (
+    <>
+      <h1>Daily Activities</h1>
+      <ul>
+        <li>
+          Day 1: <Link to="/table">Table Using Map</Link>
```
Comment (lade6501) [inline src/Daily Practice/Home/Home.jsx]:
```suggestion
          Day 1: <Link to="/tableusingmap">Table Using Map</Link>
```

File: src/App.jsx (hunk 1 above)
Comment (lmeeta) [reply src/App.jsx]: updated

**Review by lade6501 — APPROVED** [review]
Approved

**Pull request merged**